src/Security/Voter/UserVoter.php

```diff
@@ -36,58 +36,59 @@ protected function voteOnAttribute(string $attribute, $subject, TokenInterface $
         if (!$user instanceof UserInterface) {
             return false;
         }
-        if ($user->isSuperAdmin()) {
+        if ($this->security->isGranted('ROLE_ADMIN')) {
             return true;
         }
+        if ($user->isSuperAdmin()) {
+            return false;
+        }
```

Comment:
c'est bizarre ça, non ?

Reply:
Je traduit : Interdiction de toucher a un administrateur (à l'exception de si l'on est soit même un administrateur, voire condition précédente)

Reply:
ah oui ok, le $user ici n'est pas le user en cours mais celui qu'on veut changer... ok.. trop l'habitude de voir $user utilisé pour $this->decurity->getUser()